code/game/gamemodes/dynamic/dynamic_rulesets.dm

```diff
@@ -116,13 +115,12 @@
 		log_game("DYNAMIC: FAIL: [src] failed acceptable: maximum_players ([maximum_players]) < population ([population])")
 		return FALSE
 
-	if (mode.is_lategame())
-		return (flags & LATEGAME_RULESET)
```

Comment:
This will make it so that non-lategame rulesets will still trigger. They should only really trigger if there are no lategame rulesets to trigger since once we hit the late-game we want to prioritise things that will push the round towards ending.

Reply:
If we are disabling late game at low pops, then these changes will simply block any high threat lategame things from spawning, when we specifically want high threat things to spawn in the late game.

Reply:
No, that's not how that works. You would have to adjust their weights in the ruleset picking process to do so, this is simply *allowing* rulesets to executing that normally would not. Something else would have to be added if you wanted this. That is out of scope at the moment.